airflow/sensors/base_sensor_operator.py

```diff
@@ -119,9 +119,11 @@ def execute(self, context: Dict) -> Any:
                 # This gives the ability to set up non-blocking AND soft-fail sensors.
                 if self.soft_fail and not context['ti'].is_eligible_to_retry():
                     self._do_skip_downstream_tasks(context)
-                    raise AirflowSkipException('Snap. Time is OUT.')
+                    raise AirflowSkipException(
+                        "Snap. Time is OUT. {d}".format(d=self.dag.dag_id if self.has_dag() else ""))
                 else:
-                    raise AirflowSensorTimeout('Snap. Time is OUT.')
+                    raise AirflowSensorTimeout(
+                        "Snap. Time is OUT. {d}".format(d=self.dag.dag_id if self.has_dag() else ""))
```

Comment:
What would you say to extract dag id to a value and avoid duplication?
```python
log_dag_id = self.dag.dag_id if self.has_dag() else ""
if ...:
    ...
    raise AirflowSkipException(f"Snap. Time is OUT. DAG id: {log_dag_id}")
```

Reply:
Sounds good to me.